pkg/services/featuremgmt/registry.go

```diff
@@ -126,5 +126,25 @@ var (
 			Description: "Highlight Enterprise features",
 			State:       FeatureStateStable,
 		},
+		{
+			Name:        "liveDiscussions",
+			Description: "Enable Live discussions feature",
+			State:       FeatureStateAlpha,
+		},
+		{
+			Name:        "liveOrgDiscussions",
```

Comment:
I am skeptical of this flavor even behind a feature flag

Reply:
Removed